config/e2guardian/e2guardian.inc

```diff
@@ -0,0 +1,1458 @@
+		$minchildren=$matches[1];
+		$maxchildren=$matches[2];
+		}
+	else{
+		$minchildren=8;
+		$maxchildren=120;
+		}
+	if (preg_match("/(\d+)\/(\d+)/",$e2guardian['sparechildren'],$matches)){
+		$minsparechildren=$matches[1];
+		$maxsparechildren=$matches[2];
+		}
+	else{
+		$minsparechildren=8;
+		$maxsparechildren=64;
+		}
+	$maxagechildren=($e2guardian['maxagechildren']?$e2guardian['maxagechildren']:"500");
+	$maxips=($e2guardian['maxips']?$e2guardian['maxips']:"0");
+	$preforkchildren=($e2guardian['preforkchildren']?$e2guardian['preforkchildren']:"10");
+	$proxyip=($e2guardian['proxyip']?$e2guardian['proxyip']:"127.0.0.1");
+	$proxyport=($e2guardian['proxyport']?$e2guardian['proxyport']:"127.0.0.1");
+	$proxytimeout=($e2guardian['proxytimeout']?$e2guardian['proxytimeout']:"30");
+
+	#general options
+	$urlcachenumber=($e2guardian_config['urlcachenumber']?$e2guardian_config['urlcachenumber']:"1000");
+	$urlcacheage=($e2guardian_config['urlcacheage']?$e2guardian_config['urlcacheage']:"900");
+	$scancleancache=(preg_match('/scancleancache/',$e2guardian_config['scan_options'])?"on":"off");
+	$hexdecodecontent=(preg_match('/hexdecodecontent/',$e2guardian_config['scan_options'])?"on":"off");
+	$forcequicksearch=(preg_match('/forcequicksearch/',$e2guardian_config['scan_options'])?"on":"off");
+	$reverseaddresslookups=(preg_match('/reverseaddresslookups/',$e2guardian_config['scan_options'])?"on":"off");
+	$reverseclientiplookups=(preg_match('/reverseclientiplookups/',$e2guardian_config['scan_options'])?"on":"off");
+	$logclienthostnames=(preg_match('/logclienthostnames/',$e2guardian_config['scan_options'])?"on":"off");
+	$createlistcachefiles=(preg_match('/createlistcachefiles/',$e2guardian_config['scan_options'])?"on":"off");
+	$prefercachedlists=(preg_match('/prefercachedlists/',$e2guardian_config['scan_options'])?"on":"off");
+	$deletedownloadedtempfiles=(preg_match('/deletedownloadedtempfiles/',$e2guardian_config['scan_options'])?"on":"off");
+	$weightedphrasemode=($e2guardian_config['weightedphrasemode']?$e2guardian_config['weightedphrasemode']:"2");
+	$phrasefiltermode=($e2guardian_config['phrasefiltermode']?$e2guardian_config['phrasefiltermode']:"2");
+	$preservecase=($e2guardian_config['preservecase']?$e2guardian_config['preservecase']:"0");
+	$clamdscan=(preg_match('/clamdscan/',$e2guardian_config['content_scanners'])?"on":"off");
+	$icapscan=(preg_match('/icapscan/',$e2guardian_config['content_scanners'])?"on":"off");
+	$contentscannertimeout=($e2guardian_config['contentscannertimeout']?$e2guardian_config['contentscannertimeout']:"60");
+	$contentscanexceptions=($e2guardian_config['contentscanexceptions']?"on":"off");
+	$recheckreplacedurls=(preg_match('/recheckreplacedurls/',$e2guardian_config['misc_options'])?"on":"off");
+	$forwardedfor=(preg_match('/forwardedfor/',$e2guardian_config['misc_options'])?"on":"off");
+	$recheckreplacedurls=(preg_match('/icapscan/',$e2guardian_config['misc_options'])?"on":"off");
```

Comment:
This looks odd - "icapscan" result goes into $recheckreplacedurls - both of these are already on lines above.